dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -336,6 +336,22 @@ open class Jpql : JpqlDsl {
         return Paths.treat(this.toPath(), type)
     }
 
+    /**
+     * Creates an expression that represents the absolute value.
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Any, V : Number> abs(expr: KProperty1<T, @Exact V>): Expression<V> {
```

Comment:
I don't think the `ABS` is a primary function, so I think it would be better to have the function located below the `DIV`.